CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Changelog
+
+## 0.1.1
```

Comment:
Maybe version also should be bumped to 0.1.1 in `mix.exs`?

Reply:
It's a good idea. I did it.